cranelift/codegen/src/isa/aarch64/lower.isle

```diff
@@ -70,6 +70,10 @@
 (rule (lower (has_type (fits_in_64 ty) (iadd (imul x y) z)))
       (madd ty x y z))
 
+;; Fold an `isub` and `imul` combination into a `msub` instruction.
+(rule (lower (has_type (fits_in_64 ty) (isub x (def_inst (imul y z)))))
```

Comment:
The `def_inst` here is now optional, given the implicit conversions we recently added; so this pattern can be written as `(isub x (imul y z))`, which is a bit easier to read!